2021-05-19 17:13:30,428 INFO    MainThread:376636 [internal.py:wandb_internal():91] W&B internal server running at pid: 376636, started at: 2021-05-19 17:13:30.427931
2021-05-19 17:13:30,429 DEBUG   HandlerThread:376636 [handler.py:handle_request():121] handle_request: check_version
2021-05-19 17:13:30,429 INFO    WriterThread:376636 [datastore.py:open_for_write():77] open: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Weather-Clf/wandb/run-20210519_171329-2t15jmni/run-2t15jmni.wandb
2021-05-19 17:13:30,431 DEBUG   SenderThread:376636 [sender.py:send():176] send: header
2021-05-19 17:13:30,431 DEBUG   SenderThread:376636 [sender.py:send_request():186] send_request: check_version
2021-05-19 17:13:30,857 DEBUG   SenderThread:376636 [sender.py:send():176] send: run
2021-05-19 17:13:30,860 DEBUG   SenderThread:376636 [git.py:repo():33] git repository is invalid
2021-05-19 17:13:31,355 INFO    SenderThread:376636 [dir_watcher.py:__init__():168] watching files in: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Weather-Clf/wandb/run-20210519_171329-2t15jmni/files
2021-05-19 17:13:31,355 INFO    SenderThread:376636 [sender.py:_start_run_threads():699] run started: 2t15jmni with start time 1621424609
2021-05-19 17:13:31,355 DEBUG   SenderThread:376636 [sender.py:send():176] send: summary
2021-05-19 17:13:31,355 INFO    SenderThread:376636 [sender.py:_save_file():821] saving file wandb-summary.json with policy end
2021-05-19 17:13:31,356 DEBUG   HandlerThread:376636 [handler.py:handle_request():121] handle_request: run_start
2021-05-19 17:13:32,357 INFO    Thread-8  :376636 [dir_watcher.py:_on_file_created():216] file/dir created: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Weather-Clf/wandb/run-20210519_171329-2t15jmni/files/wandb-summary.json
2021-05-19 17:13:32,362 DEBUG   HandlerThread:376636 [meta.py:__init__():39] meta init
2021-05-19 17:13:32,362 DEBUG   HandlerThread:376636 [meta.py:__init__():53] meta init done
2021-05-19 17:13:32,362 DEBUG   HandlerThread:376636 [meta.py:probe():210] probe
2021-05-19 17:13:32,368 DEBUG   HandlerThread:376636 [git.py:repo():33] git repository is invalid
2021-05-19 17:13:32,368 DEBUG   HandlerThread:376636 [meta.py:_save_code():89] save code
2021-05-19 17:13:32,368 WARNING HandlerThread:376636 [meta.py:_save_code():91] unable to save code -- program entry not found
2021-05-19 17:13:32,368 DEBUG   HandlerThread:376636 [meta.py:_save_pip():57] save pip
2021-05-19 17:13:32,369 DEBUG   HandlerThread:376636 [meta.py:_save_pip():71] save pip done
2021-05-19 17:13:32,369 DEBUG   HandlerThread:376636 [meta.py:_save_conda():78] save conda
2021-05-19 17:13:33,359 INFO    Thread-8  :376636 [dir_watcher.py:_on_file_created():216] file/dir created: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Weather-Clf/wandb/run-20210519_171329-2t15jmni/files/conda-environment.yaml
2021-05-19 17:13:33,359 INFO    Thread-8  :376636 [dir_watcher.py:_on_file_created():216] file/dir created: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Weather-Clf/wandb/run-20210519_171329-2t15jmni/files/requirements.txt
